libstuff/SHTTPSManager.cpp

```diff
@@ -133,16 +131,17 @@ void SStandaloneHTTPSManager::postPoll(fd_map& fdm, SStandaloneHTTPSManager::Tra
     }
 }
 
-SStandaloneHTTPSManager::Transaction::Transaction(SStandaloneHTTPSManager& manager_) :
+SStandaloneHTTPSManager::Transaction::Transaction(SStandaloneHTTPSManager& manager_, const string& requestID) :
     s(nullptr),
     created(STimeNow()),
     finished(0),
     timeoutAt(0),
     response(0),
     manager(manager_),
     sentTime(0),
-    requestID(SThreadLogPrefix)
+    requestID(requestID.empty() ? SThreadLogPrefix : requestID)
 {
+    // TODO: Remove this block to to enable HTTPS on followers. Also the `validate` method can be removed entirely.
```

Comment:
Will that be done as part of the attached issue?

Reply:
This is step 3 in the description at the top of this PR.